src/main/java/org/testeditor/web/xtext/index/serialization/EObjectDescriptionSerialization.xtend

```diff
@@ -1,8 +1,11 @@
 package org.testeditor.web.xtext.index.serialization
 
-class EObjectDescriptionSerialization {
-	public static val EOBJECT_URI__FIELD_NAME = "eObjectURI"
-	public static val URI__FIELD_NAME = "uri"
-	public static val QUALIFIED_NAME__FIELD_NAME = "fullyQualifiedName"
-	public static val QUALIFIED_NAME__DELIMITER = "."
+/**
+ * De/Serializer constants
+ */
+interface EObjectDescriptionSerialization {
```

Comment:
Comment not needed when we name it EObjectDescriptionSerializationConstants. Should we save constants in an interface? Why not an enum?

Reply:
One could argue that these are not enumerations, putting them in either the serializer or the deserializer was not an option either, putting the in an interface makes them public static final by default ...